portal/src/views/projects/ProjectPublic.vue

```diff
@@ -52,14 +50,95 @@
                         :src="module.url"
                     />
                 </div>
+                <div class="right-actions">
+                    <button class="button-social">
+                        <img src="@/assets/icon-heart-dark-blue.svg" width="16px" alt="Icon">
+                        Follow
+                    </button>
+                    <button class="button-social">
+                        <img src="@/assets/icon-share.svg" width="12px" alt="Icon">
+                        Share
+                    </button>
+                </div>
             </div>
         </div>
 
+        <div class="update">
+            <UserPhoto :user="user" />
+            <div>
+                <h1> Project update title </h1>
+                <h2> by Lauren Martin | 01/21/2020 </h2>
+                <p>
+                    Thank you so much for contributing and supporting us during this project. Lorem ipsum dolor sit amet,
```

Comment:
Looks like a hard coded project update?

Reply:
I usually componentize once the data flow is decided.
Couldn't find the needed data in the parent component or a separate api.